analytics/analytics-ui/src/app/results/results.component.html

```diff
@@ -0,0 +1,18 @@
+<div style="margin-left:110px">
+
+  <!--Title-->
+  <h1>Results</h1>
+
+  <!--Back button-->
+  <button mat-raised-button id="back-params" [routerLink]="['..']" style="cursor:pointer"
```

Comment:
You should use the <a> tag for links. https://material.angular.io/components/button/examples